Tasks/IISWebAppDeployment/MsDeployOnTargetMachines.ps1

```diff
@@ -343,6 +347,26 @@ function Create-WebSite
     Run-Command -command $command
 }
 
+function Run-AdditionalCommands
```

Comment:
I thought the task input said these are additional arguments to appcmd and not additional commands itself. It appears we are now two commands, the main command once and with additional arguments once. Is that right way to do?